backend/src/main/java/com/woowacourse/friendogly/club/service/ClubCommandService.java

```diff
@@ -1,11 +1,87 @@
 package com.woowacourse.friendogly.club.service;
 
 
+import com.woowacourse.friendogly.club.domain.Club;
+import com.woowacourse.friendogly.club.dto.request.SaveClubMemberRequest;
+import com.woowacourse.friendogly.club.dto.request.SaveClubRequest;
+import com.woowacourse.friendogly.club.dto.response.SaveClubMemberResponse;
+import com.woowacourse.friendogly.club.dto.response.SaveClubResponse;
+import com.woowacourse.friendogly.club.repository.ClubMemberRepository;
+import com.woowacourse.friendogly.club.repository.ClubRepository;
+import com.woowacourse.friendogly.exception.FriendoglyException;
+import com.woowacourse.friendogly.member.domain.Member;
+import com.woowacourse.friendogly.member.repository.MemberRepository;
+import com.woowacourse.friendogly.pet.domain.Pet;
+import com.woowacourse.friendogly.pet.repository.PetRepository;
+import java.util.List;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 @Service
 @Transactional
 public class ClubCommandService {
 
+    private final ClubRepository clubRepository;
+    private final MemberRepository memberRepository;
+    private final PetRepository petRepository;
+
+    public ClubCommandService(
+            ClubRepository clubRepository,
+            MemberRepository memberRepository,
+            PetRepository petRepository,
+            ClubMemberRepository clubMemberRepository) {
+        this.clubRepository = clubRepository;
+        this.memberRepository = memberRepository;
+        this.petRepository = petRepository;
+    }
```

Comment:
```suggestion
    public ClubCommandService(
            ClubRepository clubRepository,
            MemberRepository memberRepository,
            PetRepository petRepository,
            ClubMemberRepository clubMemberRepository
    ) {
        this.clubRepository = clubRepository;
        this.memberRepository = memberRepository;
        this.petRepository = petRepository;
    }
```

1. 개행 바꿔주세요~~
2. `ClubMemberRepository`는 어떻게 주입받고 있나요?

Reply:
연관 관계 리팩터링 하면서 더 이상 ClubMember~ ClubPet~ repository를 사용할 필요성이 없어졌는데, 까먹고 안 지웠네요